examples/azure-ark/porter.yaml

```diff
@@ -0,0 +1,82 @@
+mixins:
+  - azure
+  - helm
+ 
+name: porter-azure-ark
+description: "A Porter example using Azure Storage and Ark"
+version: 0.1.0
+# TODO: change to deislabs
+invocationImage: quay.io/vdice/porter-azure-ark:latest
+
+credentials:
+- name: SUBSCRIPTION_ID
+  env: AZURE_SUBSCRIPTION_ID
+- name: TENANT_ID
+  env: AZURE_TENANT_ID
+- name: CLIENT_ID
+  env: AZURE_CLIENT_ID
+- name: CLIENT_SECRET
+  env: AZURE_CLIENT_SECRET
+- name: RESOURCE_GROUP
+  env: AZURE_RESOURCE_GROUP
+- name: kubeconfig
+  path: /root/.kube/config
+
+parameters:
+- name: storage_account_name
+  type: string
+  default: porterdemoaccount
+
+- name: storage_container_name
+  type: string
+  default: porterdemocontainer
+
+install:
+  - description: "Create Azure Storage Account and Container"
+    azure:
+      type: storage
+      name: porter-azure-ark
+      resourceGroup:
+        source: bundle.credentials.RESOURCE_GROUP
+      parameters:
+        location: "eastus"
```

Comment:
Will this run into trouble if the resource group isn't in eastus?

Reply:
Ah, good point; I'd imagine it would.  Perhaps we parameterize this value?  If so, should this parameter be non-required, with a default of `eastus` (but changeable) or required with no default?

Reply:
+1 default to eastus or soemthing.  Out of curiosity, why the credentials instead of a parameter to the bundle? they seem like conceptually different things.

Reply:
Mmm, good call -- indeed, resource group _should_ be a parameter.  Will fixup as well.

Reply:
👍 Seems like a good way to do it, a parameter with a default.

Reply:
Now updated; PTAL